app/assets/stylesheets/components/_expander.scss

```diff
@@ -87,3 +88,7 @@
     display: block;
   }
 }
+
+.app-c-option-select__icon {
```

Comment:
option-select styles should go in the options-select partial. this file is the expander partial

half pixels should be avoided as some browser round them up, some round them down

Reply:
Moved this into the option-select partial, and changed it to 3px